cob_bms_driver/src/fake_bms.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+import rospy
+from cob_msgs.msg import PowerState
+from cob_srvs.srv import SetFloat, SetFloatResponse
```

Comment:
please add 
```
<exec_depend>cob_msgs</exec_depend>
<exec_depend>cob_srvs</exec_depend>
```
to `cob_bms_driver/package.xml` (respect alphabetical order :wink:)